custom.scss

```diff
@@ -992,7 +992,7 @@ body.listing-page>.content {
   content: "";
   display: inline-block;
   margin-right: 4px;
-  background: url(%%flair%%) -9px -1319px no-repeat;
+  background: url(%%flair-general%%) -9px -183px no-repeat;
```

Comment:
This took me a second to puzzle out. I guess when they did the flair originally, they didn't base it on the 20x40 that normal flair is, so -9px is where the image starts when it's centered in a 40px wide space. The 183 is because that's where it starts vertically.